src/lib/y2security/selinux.rb

```diff
@@ -168,20 +261,17 @@ def find_mode(id)
     #
     # @return [Mode] disabled or found SELinux mode
     def proposed_mode
-      id = Yast::ProductFeatures.GetFeature("globals", "selinux_mode").to_sym
+      id = product_feature_settings[:mode]
+      found_mode = Mode.find(id)
 
-      log.info("Proposing `#{id}` SELinux mode.")
+      if found_mode.nil?
+        log.info("Proposed SELinux mode `#{id}` not found.")
 
-      find_mode(id)
-    end
+        return
```

Comment:
Is there the return value missed ? Or do I have missed something ? :-)

Reply:
Not missing, returning `nil`